scripts/install-deps.sh

```diff
@@ -666,13 +668,13 @@ do_install_shellcheck() {
             return $ret_val
         fi
 
-        ln -sf "${HOME}/.cabal/bin/shellcheck" "${work_folder}/deviceupdate-shellcheck" || return 1
+        ln -sf "${HOME}/.cabal/bin/shellcheck" "/tmp/deviceupdate-shellcheck" || return 1
```

Comment:
previously we had an alias for the target folder.  now you're hardcoding /tmp here and on 677